authors/apps/authentication/tests/test_jwt.py

```diff
@@ -0,0 +1,176 @@
+from django.test import TestCase, Client
+from ..models import User
+import json
+import jwt
+import authors.settings
+from django.conf import settings
+from datetime import datetime, timedelta
+
+
+class TestJWT(TestCase):
+
+    def setUp(self):
+        """ Funtion to setup some code that will be needed for unittests """
+        self.email = 'boomboom@gmail.com'
+        self.username = 'testing12'
+        self.password = 'testuserpass'
+
+        # create a user that will be logged in
+        self.user = User.objects.create_user(
+            self.username, self.email, self.password)
+
+        # verify a user's account and save
+        self.user.is_account_verfied = True
+        self.user.save()
+
+        self.data = {
+            'user': {
+                'username': self.username,
+                'email': self.email,
+                'password': self.password,
+            }
+        }
+
+        self.test_client = Client()
+
+    def tearDown(self):
+        pass
+
+    def login_a_user(self):
+        """
+        Reusable function to login a user
+        """
+
+        response = self.test_client.post(
+            "/api/users/login/", data=json.dumps(self.data), content_type='application/json')
+        token = response.json()['user']['token']
+        return token
+
+        # return (headers = {'HTTP_AUTHORIZATION': 'Token ' + token })
```

Comment:
@lytes20 
- We should remove uncommented code

Reply:
yes, yes, let me work on this now